Documentation/devel/release.md

```diff
@@ -34,33 +34,33 @@ The rkt version is [hardcoded in the repository](https://github.com/coreos/rkt/b
 After merging and going back to master branch, we check out the release version and tag it:
 
 - `git checkout HEAD^` should work; sanity check configure.ac (2nd line) after doing this
-- Build rkt, we'll use this in a minute:
-  - `git clean -ffdx && ./autogen.sh && ./configure --enable-tpm=no && make BUILDDIR=release-build -j 4`
-    - This will build the `coreos`, `kvm` and `fly` flavors and make `coreos` the default
-    - Use make's `-j` parameter as you see fit
+- Build rkt inside rkt (so make sure you have rkt in your $PATH), we'll use this in a minute:
+  - `git clean -ffdx && sudo ./scripts/acbuild-rkt-builder.sh`
```

Comment:
I guess we should recommend using separate rkt git repo, because rir was creating some files owned by root outside the builddir too.